Makefile

```diff
@@ -70,7 +70,7 @@ integration-win: binaries/client build-extension-types build-extensions
 .PHONY: build
 build: node_modules binaries/client
 	yarn run npm:fix-build-version
-	$(MAKE) build-extensions
+	$(MAKE) build-extensions -B
```

Comment:
This is required for local `make build`'s so that extensions are actually loaded.